src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private int variable;
+    private int age;
+    private int size;
+
+    public static void main(String[] args) {
+        Dog dog = new Dog();
+        Cat cat = new Cat();
+    }
+
+    private String getFromOneToTen() {
+        String result;
+        result = "";
+        for (int i = 0; i < 10; i++) {
+            if (i % 2 == 0) {
+                result = result + i + " ";
+            }
+        }
+        return result;
+    }
+
+    private void sayHello(String hello) {
+        System.out.println("Hello, Mate! I am happy to see you! This class looks weird!"
+                +
+                " Your task is to fix the checkstyle in this file."
+                +
+                " Your goal is to run `mvn clean package` and receive `BUILD SUCCESS` message");
```

Comment:
Same here about `+` operator, it should come at the beginning of each line
[Java Style Doc](https://mate-academy.github.io/style-guides/java/java.html#s4.5-line-wrapping)